src/main/java/uk/gov/companieshouse/pscstatement/delta/mapper/StatementMapper.java

```diff
@@ -19,6 +21,14 @@
 public interface StatementMapper {
 
     MapperUtils mapperUtils = new MapperUtils();
+    Map<String, String> restrictionsNoticeWithdrawalReasons = Map.ofEntries(
+            Map.entry("1", "restrictions-notice-withdrawn-by-company"),
+            Map.entry("2", "restrictions-notice-withdrawn-by-court-order"),
+            Map.entry("3", "restrictions-notice-withdrawn-by-lp"),
+            Map.entry("4", "restrictions-notice-withdrawn-by-court-order-lp"),
+            Map.entry("5", "restrictions-notice-withdrawn-by-partnership"),
+            Map.entry("6", "restrictions-notice-withdrawn-by-court-order-p")
+    );
```

Comment:
Does this need to be a global config? We could hide the creation behind the null check and then we only instantiate a new hashmap when necessary.